include/envoy/stats/stats.h

```diff
@@ -225,41 +225,60 @@ class ParentHistogram : public virtual Histogram {
 typedef std::shared_ptr<ParentHistogram> ParentHistogramSharedPtr;
 
 /**
- * A sink for stats. Each sink is responsible for writing stats to a backing store.
+ * Provides cached access to a particular store's stats.
  */
-class Sink {
+class StatsSource {
```

Comment:
Should this be named Source instead of StatsSource? It's already in namespace Stats.